meta-resin-common/recipes-support/resin-supervisor/files/resin-supervisor.service

```diff
@@ -25,6 +25,7 @@ ExecStart=@BASE_BINDIR@/bash -c 'source @SBINDIR@/resin-vars && \
     -v /proc/net/fib_trie:/mnt/fib_trie \
     -v /var/log/supervisor-log:/var/log \
     -v /:/mnt/root \
+    -e DOCKER_ROOT=/mnt/root/var/lib/docker \
```

Comment:
I'm not sure who would set /var/lib/rce as default. Can you elaborate where this is coming from? We should fix it at the source, not here